.gitignore

```diff
@@ -4,6 +4,10 @@
 /dist
 /tmp
 /out-tsc
+
+# diagrams output
+out/docs/*
```

Comment:
la nostra convenzione è di mettere i diagrammi in cartelle "diagrams" con l'output in una sottocartella "img"

```suggestion
# diagrams output
**/diagrams/img/**
```